internal/repository/settings/aggregate.go

```diff
@@ -0,0 +1,23 @@
+package settings
+
+import "github.com/zitadel/zitadel/internal/eventstore"
+
+const (
+	AggregateType    = "settings"
```

Comment:
Maybe quickly to discuss: Do we want to have a `settings` or an `organization_settings` aggregate?
@adlerhurst what's your opinion.